src/main.rs

```diff
@@ -287,6 +296,26 @@ fn settings() -> Result<Settings, Failure> {
         |path| Some(PathBuf::from(path)),
     );
 
+    // Read the config file path.
+    let default_output_path = toastfile_path.parent().unwrap().to_path_buf();
+    let output_path = matches.value_of(CONFIG_OUTPUT_DIR_OPTION).map_or_else(
+        || Ok(default_output_path),
+        |path| {
+            let candidate_path = PathBuf::from(path);
+            return if candidate_path.exists() && candidate_path.is_dir() {
```

Comment:
You can remove `return`.

I would remove this check entirely. There may be other issues with the output path besides not being a directory (e.g., not being writable). It's not worth trying to catch these issues in advance—any problems will be reported already by `docker::copy_from_container`.